Microsoft.SystemForCrossDomainIdentityManagement/Service/BulkCreationOperationState.cs

```diff
@@ -0,0 +1,268 @@
+
+                        PatchOperation2Combined patchOperation =
+                            PatchOperation2Combined.Create(
+                                OperationName.Add,
+                                AttributeNames.Manager,
+                                user.EnterpriseExtension.Manager.Value);
+                        PatchRequest2 patchRequest = new PatchRequest2();
+                        patchRequest.AddOperation(patchOperation);
+
+                        this.AddSubordinate(patchResourceIdentifier, patchRequest, context);
+
+                        user.EnterpriseExtension.Manager = null;
+                    }
+                }
+
+                if (operationData.IsResourceType(SchemaIdentifiers.Core2Group))
+                {
+                    GroupBase group = operationDataJson.ToObject<Core2Group>();
+                    resource = group;
+                    if (group.Members != null && group.Members.Any())
+                    {
+                        string resourceIdentifier =
+                            string.Format(
+                                CultureInfo.InvariantCulture,
+                                BulkCreationOperationState.RelativeResourceIdentifierTemplate,
+                                ProtocolConstants.PathGroups,
+                                this.Operation.Identifier);
+                        Uri patchResourceIdentifier =
+                            new Uri(resourceIdentifier, UriKind.Relative);
+
+                        PatchRequest2 patchRequest = new PatchRequest2();
+                        foreach (Member member in group.Members)
+                        {
+                            if (member == null || string.IsNullOrWhiteSpace(member.Value))
+                            {
+                                continue;
+                            }
+
+                            string memberValue = System.Text.Json.JsonSerializer.Serialize(member);
+                            if (!string.IsNullOrWhiteSpace(memberValue))
+                            {
+                                PatchOperation2Combined patchOperation =
+                                    PatchOperation2Combined.Create(
+                                        OperationName.Add,
+                                        AttributeNames.Members,
+                                        memberValue);
+                                patchRequest.AddOperation(patchOperation);
+                            }
+                        }
+
+                        this.AddSubordinate(patchResourceIdentifier, patchRequest, context);
+
+                        group.Members = null;
+                    }
+                }
+                if (null == resource)
+                {
+                    string invalidOperationExceptionMessage =
+                                                        string.Format(
+                                                            CultureInfo.InvariantCulture,
```

Comment:
Consider a smaller indent, to be one indent in from string invalid ... . Lines that don't extend too far right (beyond the 135 character margin) are easier for the sight-impaired to read with screen manifiers.